TrkHitReco/src/CombineStrawHits_module.cc

```diff
@@ -228,6 +231,15 @@ namespace mu2e {
     combohit._tres       = _terr/sqrt(combohit._nsh); // error proportional to # of straws (roughly)
     //      float wvar           = sqrtf((wacc2/weights-wacc/weights*wacc/weights));//define quality as variance/average ratio
     combohit._qual       = 1.0; // quality isn't used and should be removed FIXME
+//-----------------------------------------------------------------------------
+// if combohit._wres is not consistent with the spread of the individual hits,
+// increase it
+//-----------------------------------------------------------------------------
+    if (_checkWres) {
+      float wdist2 = wacc2/weights;
+      float sigw   = sqrt((wdist2-combohit._wdist*combohit._wdist)/(combohit.nCombo()-1+1e-12));
+      if (combohit._wres < sigw/2) combohit._wres = sigw;
```

Comment:
What is the '2' here?  It looks like a tuned parameter.  If so, it should be a fcl parameter.  Making a statistical consistency check makes sense, but this seems an ad-hoc algorithm.  How about using the PDG formula for how errors are expanded when measurements are inconsistent?  That's well-documented and has a smooth behavior.  Here's an interesting paper on another technique: https://arxiv.org/pdf/2004.01219.pdf